src/main/java/pl/piotrpawel/biuropodrozy/Trip.java

```diff
@@ -0,0 +1,145 @@
+package pl.piotrpawel.biuropodrozy;
+
+
+import pl.piotrpawel.biuropodrozy.config.TripType;
+
+import java.time.LocalDate;
+
+public class Trip {
+
+    String travelFrom; // miasto, lotnisko
+    String travelTo; // miasto, hotel, lotnisko
```

Comment:
proponuję się skupić np. na mieście wylotu i powrotu, żeby było:
City travelFrom
City travelTo
Pozwoli to ograniczyć problemy związane z typowymi pomyłkami podczas konfigurowania wycieczek, takimi jak literówki w nazwie miasta itp. Dzięki temu będzie w bazie lista miast spośród których będzie dokonywany wybór.